### ISSUE TITLE
minecraft.jar crashes, WURST

### ISSUE THREAD

**Issue by Soundisimore** [Wurst-Imperium/Wurst-Client#2192]
```
---- Minecraft Crash Report ----
// Why did you do that?

Time: 9/10/15 3:32 PM
Description: Initializing game

java.lang.NumberFormatException: For input string: "0�40.0"
	at java.lang.NumberFormatException.forInputString(NumberFormatException.java:65)
	at java.lang.Integer.parseInt(Integer.java:580)
	at java.lang.Integer.valueOf(Integer.java:766)
	at tk.wurst_client.files.FileManager.loadSliders(FileManager.java:369)
	at tk.wurst_client.gui.GuiManager.setup(GuiManager.java:197)
	at tk.wurst_client.Client.startClient(Client.java:46)
	at bsu.aj(Minecraft.java:588)
	at bsu.a(Minecraft.java:401)
	at net.minecraft.client.main.Main.main(SourceFile:120)


A detailed walkthrough of the error, its code path and all known details is as follows:
---------------------------------------------------------------------------------------

-- Head --
Stacktrace:
	at java.lang.NumberFormatException.forInputString(NumberFormatException.java:65)
	at java.lang.Integer.parseInt(Integer.java:580)
	at java.lang.Integer.valueOf(Integer.java:766)
	at tk.wurst_client.files.FileManager.loadSliders(FileManager.java:369)
	at tk.wurst_client.gui.GuiManager.setup(GuiManager.java:197)
	at tk.wurst_client.Client.startClient(Client.java:46)
	at bsu.aj(Minecraft.java:588)

-- Initialization --
Details:
Stacktrace:
	at bsu.a(Minecraft.java:401)
	at net.minecraft.client.main.Main.main(SourceFile:120)

-- System Details --
Details:
	Minecraft Version: 1.8
	Operating System: Mac OS X (x86_64) version 10.9.5
	Java Version: 1.8.0_60, Oracle Corporation
	Java VM Version: Java HotSpot(TM) 64-Bit Server VM (mixed mode), Oracle Corporation
	Memory: 52314064 bytes (49 MB) / 241967104 bytes (230 MB) up to 1060372480 bytes (1011 MB)
	JVM Flags: 5 total; -Xmx1G -XX:+UseConcMarkSweepGC -XX:+CMSIncrementalMode -XX:-UseAdaptiveSizePolicy -Xmn128M
	IntCache: cache: 0, tcache: 0, allocated: 0, tallocated: 0
	Launched Version: Wurst
	LWJGL: 2.9.2
	OpenGL: Intel Iris OpenGL Engine GL version 2.1 INTEL-8.28.32, Intel Inc.
	GL Caps: Using GL 1.3 multitexturing.
Using GL 1.3 texture combiners.
Using framebuffer objects because ARB_framebuffer_object is supported and separate blending is supported.
Shaders are available because OpenGL 2.1 is supported.
VBOs are available because OpenGL 1.5 is supported.

	Using VBOs: No
	Is Modded: Definitely; Client brand changed to 'Wurst'
	Type: Client (map_client.txt)
	Resource Packs: []
	Current Language: English (US)
	Profiler Position: N/A (disabled)
```

**Comment by mcccs** [comment]
```
- Minecraft Crash Report ----
// Why did you do that?

Time: 9/10/15 3:32 PM
Description: Initializing game

java.lang.NumberFormatException: For input string: "0�40.0"
at java.lang.NumberFormatException.forInputString(NumberFormatException.java:65)
at java.lang.Integer.parseInt(Integer.java:580)
at java.lang.Integer.valueOf(Integer.java:766)
at tk.wurst_client.files.FileManager.loadSliders(FileManager.java:369)
at tk.wurst_client.gui.GuiManager.setup(GuiManager.java:197)
at tk.wurst_client.Client.startClient(Client.java:46)
at bsu.aj(Minecraft.java:588)
at bsu.a(Minecraft.java:401)
at net.minecraft.client.main.Main.main(SourceFile:120)

A detailed walkthrough of the error, its code path and all known details is as follows:

-- Head --
Stacktrace:
at java.lang.NumberFormatException.forInputString(NumberFormatException.java:65)
at java.lang.Integer.parseInt(Integer.java:580)
at java.lang.Integer.valueOf(Integer.java:766)
at tk.wurst_client.files.FileManager.loadSliders(FileManager.java:369)
at tk.wurst_client.gui.GuiManager.setup(GuiManager.java:197)
at tk.wurst_client.Client.startClient(Client.java:46)
at bsu.aj(Minecraft.java:588)

-- Initialization --
Details:
Stacktrace:
at bsu.a(Minecraft.java:401)
at net.minecraft.client.main.Main.main(SourceFile:120)

-- System Details --
Details:
Minecraft Version: 1.8
Operating System: Mac OS X (x86_64) version 10.9.5
Java Version: 1.8.0_60, Oracle Corporation
Java VM Version: Java HotSpot(TM) 64-Bit Server VM (mixed mode), Oracle Corporation
Memory: 52314064 bytes (49 MB) / 241967104 bytes (230 MB) up to 1060372480 bytes (1011 MB)
JVM Flags: 5 total; -Xmx1G -XX:+UseConcMarkSweepGC -XX:+CMSIncrementalMode -XX:-UseAdaptiveSizePolicy -Xmn128M
IntCache: cache: 0, tcache: 0, allocated: 0, tallocated: 0
Launched Version: Wurst
LWJGL: 2.9.2
OpenGL: Intel Iris OpenGL Engine GL version 2.1 INTEL-8.28.32, Intel Inc.
GL Caps: Using GL 1.3 multitexturing.
Using GL 1.3 texture combiners.
Using framebuffer objects because ARB_framebuffer_object is supported and separate blending is supported.
Shaders are available because OpenGL 2.1 is supported.
VBOs are available because OpenGL 1.5 is supported.

Using VBOs: No Is Modded: Definitely; Client brand changed to 'Wurst' Type: Client (map_client.txt) Resource Packs: [] Current Language: English (US) Profiler Position: N/A (disabled)
```

**Comment by Alexander01998** [comment]
It seems that you slider values  have been corrupted. Delete `%appdata%\.minecraft\wurst\sliders.json` and it should be fine.
